11-foundations-randomization.qmd

```diff
@@ -565,8 +565,7 @@ What would be the expected difference between the proportions of "not buy video"
 
 ------------------------------------------------------------------------
 
-Since the simulated groups are of equal size, we would expect $53 / 2 = 26.5,$ i.e., 26 or 27, "not buy video" cards in each simulated group, yielding a simulated point estimate of the difference in proportions of 0% .
-However, due to random chance, we might also expect to sometimes observe a number a little above or below 26 and 27.
+Since the simulated groups are of equal size, we would expect $53 / 2 = 26.5,$ i.e., 26 or 27, "not buy video" cards in each simulated group, yielding a simulated point estimate of the difference in proportions of 0%. However, due to random chance, we might also expect to sometimes observe a number a little above or below 26 and 27.
```

Comment:
```suggestion
Since the simulated groups are of equal size, we would expect $53 / 2 = 26.5,$ i.e., 26 or 27, "not buy video" cards in each simulated group, yielding a simulated point estimate of the difference in proportions of 0%. 
However, due to random chance, we might also expect to sometimes observe a number a little above or below 26 and 27.
```